AmplifyPlugins/DataStore/Tests/DataStoreHostApp/AWSDataStorePluginCPKTests/PrimaryKey/AWSDataStorePrimaryKeyBaseTest.swift

```diff
@@ -177,7 +177,8 @@ extension AWSDataStorePrimaryKeyBaseTest {
     ///   - onFailure: failure callback
     func assertMutationsParentChild<P: Model & ModelIdentifiable,
```

Comment:
suggestion : can this be renamed to `assertCreateMutation` and have different functions for `create` and `delete` assertions instead of passing a `Bool` `shouldDeleteParent` ?